src/main/java/uk/gov/digital/ho/hocs/info/api/dto/SchemaDto.java

```diff
@@ -4,13 +4,15 @@
 import lombok.AccessLevel;
```

Comment:
Please see the comment below, if you choose to leave your change though, can you remove this import please?

Reply:
I've removed this change.